### PR TITLE
added functionality for rating tutorials via google forms

### PR DESCRIPTION
**Pull request by guptadiksha307**

### CODE REVIEW

File: content/tutorials/index.md [hunk 1]
```diff
@@ -29,3 +29,9 @@ Here's a short list of video tutorials produced by community members. If you'd l
 - [YeastMine videos](https://www.youtube.com/watch?v=fnWv6qRl_DA&list=PL0VHJdmmIuj-b00aNRfqwMe9TvkfWWcyZ) - SGD has quite a broad range of tutorials based on S. Cerevisiae data.
 - [MouseMine at MGI](https://www.youtube.com/watch?v=FtlsoM8TGGs&t=139s) - M. musculus InterMine intro
 - [Python tutorials](https://mybinder.org/v2/gh/intermine/intermine-ws-python-docs/11613565708ef555a34a11f5856907ca2d2856e3) - Tutorials to help you get familiarised with the intermine-python package. The tutorials are in the form of Jupyter-Notebooks.
+
+## Rating for tutorials
+
+submit your feedbacks regarding the tutorials/videos you have just completed, including feedback on video structure, content, and instructor using this Google form!
+
+- [Tutorial Evaluation](https://docs.google.com/forms/d/e/1FAIpQLSe4sJbmWfzBGYQRObbDvP7_thCCkzglikOsbjmuRyhZC5aEKw/viewform?usp=sf_link)
```
Comment (Persdre) [inline content/tutorials/index.md]: A bit of small advice, maybe you can add some thank you words in the form haha. If I am a learner, I would feel happy if see them when I voluntarily finished the form!

Comment (guptadiksha307) [reply content/tutorials/index.md]: Hi @Persdre I have updated the google form.
image
please review.!

File: content/tutorials/index.md [hunk 2]
```diff
@@ -29,3 +29,9 @@ Here's a short list of video tutorials produced by community members. If you'd l
 - [YeastMine videos](https://www.youtube.com/watch?v=fnWv6qRl_DA&list=PL0VHJdmmIuj-b00aNRfqwMe9TvkfWWcyZ) - SGD has quite a broad range of tutorials based on S. Cerevisiae data.
 - [MouseMine at MGI](https://www.youtube.com/watch?v=FtlsoM8TGGs&t=139s) - M. musculus InterMine intro
 - [Python tutorials](https://mybinder.org/v2/gh/intermine/intermine-ws-python-docs/11613565708ef555a34a11f5856907ca2d2856e3) - Tutorials to help you get familiarised with the intermine-python package. The tutorials are in the form of Jupyter-Notebooks.
+
+## Rating for tutorials
+
+submit your feedbacks regarding the tutorials/videos you have just completed, including feedback on video structure, content, and instructor using this Google form!
```
Comment (yochannah) [inline content/tutorials/index.md]: Capital S for the start of the sentence please :) ⭐️

File: content/tutorials/index.md (hunk 1 above)
Comment (yochannah) [inline content/tutorials/index.md]: The form is great! Could we add it as an embedded form rather than as a link out?

**Comment by guptadiksha307** [comment]
> @guptadiksha307 This is a really great start! I've added a couple of comments - could you address these and let me know when you're done? <3 Also worth thinking about where the responses to this form go - will you be able to pass ownership of the form to an intermine team member once this is merged? :)

@yochannah This google form is linked with the google spreadsheet. Currently, I'm the owner of the google response sheet but there is an option to pass on the ownership to someone else so that he/she can view or edit the whole record. So yes, I can pass ownership of the form to someone else if you provide me his/her email id :)

**Comment by guptadiksha307** [comment]
@yochannah I have embedded the google form also. Please review.

**Comment by uosl** [comment]
I'm sorry that we didn't follow up with your PR! It will be difficult for us to pick this up again as it's been a while since, so I'll have to close it.

**Pull request closed without merging**